src/index.js

```diff
@@ -41,18 +41,29 @@ export default class Form extends React.Component {
   floatingLabelFocusStyle = this.props.focusStyle || {};
 
   handleEmojiText = (input) => {
-    var emoji = new EmojiConvertor();
-    return emoji.replace_colons(input);
+    if(_.isString(input)) {
+      const emoji = new EmojiConvertor();
+      return emoji.replace_colons(input);
+      }
```

Comment:
Soon or later we need to introduce the prettier here as well. :)

Reply:
Very good point.